Answers.md

```diff
@@ -1,9 +1,15 @@
-1. If you had to describe semantic HTML to the next cohort of students, what would you say?
-
-2. Describe some differences between ```display: block;``` and ```display: inline;```.
-
-3. What are the 4 areas of the box model?
-
-4. While using flexbox, what axis are you using when you use the property: ```align-items: center```?
-
-5. What is the git command to commit staged changes as well as write a message? 
```

Comment:
On the next Sprint Challenge and going forward, please provide answers to the questions. You are welcome to use your resources to compile answers, but answer in your own words.